### ISSUE TITLE
Make list show folder sizes

### ISSUE THREAD

**Issue by Wilker-a** [zhanghai/MaterialFiles#331]
maybe not by default, just in case there may be too many files to process, but it would be useful to have folder sizes shown right away like the file sizes are currently show on the lists right now. :3

**Comment by bbigam** [comment]
I would be more interested in seeing the number of items in each folder in the list view. Lots of other files managers do this and it doesn't seem to be resource intensive to do so.

Here's how it looks in Simple File Manager: https://raw.githubusercontent.com/SimpleMobileTools/Simple-File-Manager/master/fastlane/metadata/android/en-US/images/phoneScreenshots/app_1.jpg

**Comment by opusforlife2** [comment]
Possible info to show: number of items, size, 'last modified' date.

@zhanghai Could you edit the title to make it more broad?

**Comment by decodering22** [comment]
Yes please add the number of items in folder to help identify empty folders.

**Comment by decodering22** [comment]
It seems you have no intention of adding this feature to improve the app? It has been clearly requested multiple times. Why the aversion?

**Comment by joelchrono12** [comment]
I would like to have folder size showing up, it has always been really useful for me when I used to use Solid Explorer. But since I decided to go Foss it has been the only thing I am missing...
